src/ui/reducers/network.ts

```diff
@@ -22,6 +23,7 @@ export type NetworkState = {
   requestBodies: Record<string, RequestBodyData[]>;
   requests: RequestInfo[];
   selectedRequestId: string | null;
+  copyCUrlAble: Record<RequestId, boolean>;
```

Comment:
Why is this named "CUrlAble" ? Seems odd naming.

Reply:
I will modify it to be more appropriate.

Reply:
That's okay. No need.